src/Components/Clock.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+type Props = {
+  name: string,
+};
+export class Clock extends React.Component<Props> {
+  state = {
+    today: new Date(),
+  };
+
+  timerId = 0;
+
+  componentDidMount() {
+    this.timerId = window.setInterval(() => {
+      this.setState({
+        today: new Date(),
+      });
+      // eslint-disable-next-line no-console
+      console.info(this.state.today.toUTCString()
+        .slice(-12, -4));
```

Comment:
you do this formating twice, I'd create a separate function for it

Reply:
Good point,  added normalize date function,